integrations/sources/apache-iceberg.mdx

```diff
@@ -32,6 +32,7 @@ You don’t need to specify the column name for the Iceberg source, as RisingWav
 | s3.region      | Optional. The region where the S3 bucket is hosted. Either `s3.endpoint` or `s3.region` must be specified.                                                                                              |
 | s3.access.key  | Required. Access key of the S3 compatible object store.                                                                                                                                             |
 | s3.secret.key  | Required. Secret key of the S3 compatible object store.                                                                                                                                             |
+| s3.path.style.access | Optional. Determines the access style for S3. If `true`, use path-style; if `false`, use [virtual-hosted–style](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html). |
```

Comment:
Hello @chenzl25 , wanna double check this parameter applies to both source and sink, right?

Reply:
Yes